examples/apache-directory-list-local/webdav.conf

```diff
@@ -1,23 +1,16 @@
-ServerName webdav.server.com
-DocumentRoot /srv/webdav/root
+<VirtualHost *:80>
+    ServerName localhost
+    DocumentRoot /var/www/html/
 
-# This prevents indexes from being parsed (unless they're called the below...)
-AccessFileName .file-that-will-never-exist
-DirectoryIndex .file-that-will-never-exist
```

Comment:
I think we should keep this as a comment, with a note saying `for Apache < 2.4` or something, so that this can still serve as an example for anyone running an older version.

Reply:
A good point!